app/src/pages/outside/registrationPage/registrationPageContainer.jsx

```diff
@@ -90,13 +105,30 @@ export class RegistrationPageContainer extends Component {
 
   render() {
     const uuid = this.props.uuid;
-    return !uuid || this.state.isLoadingFinished ? (
-      <RegistrationPage
-        tokenProvided={Boolean(uuid)}
-        tokenActive={this.state.isTokenActive}
-        email={this.state.email}
-        onRegistrationSubmit={this.registrationHandler}
-      />
-    ) : null;
+    const extensions = this.props.extensions;
+    const type = this.props.pageProps.type;
```

Comment:
better use destructuring `const {extensions, pageProps: {type}} = this.props;`
also pageProps I guess should be call `query` as it is standard name for `?type=something`